templates/documentation.html

```diff
@@ -0,0 +1,297 @@
+      <div class="row">
+        <div class="col-lg-6">
+          <div class="row centered">
+            <span class="section-title">Getting help</span>
+            <div class="col-lg-12">
+              <div class="row">
+                <div class="col-sm-6">
+                  <h2><i class="fa fa-keyboard-o"></i> <a href="https://gitter.im/inveniosoftware/invenio">Chat</a>
+                  </h2>
+                  <p>
+                    Chat with Invenio developers
+                  </p>
+                </div>
+                <div class="col-sm-6">
+                  <h2><i class="fa fa-comment"></i> <a href="https://invenio-talk.web.cern.ch/">Forum</a></h2>
+                  <p>
+                    Ask questions on the forum
+                  </p>
+                </div>
+              </div>
+              <div class="row">
+                <div class="col-sm-6">
+                  <h2><i class="fa fa-users"></i> <a href="{{ '/people'|url }}">People</a></h2>
+                  <p>
+                    Review the list of contributors
+                  </p>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+        <div class="col-lg-6">
+          <div class="row centered">
+            <span class="section-title">Contributing</span>
+            <div class="col-lg-12">
+              <div class="row">
+                <div class="col-sm-6">
+                  <h2>
+                    <i class="fa fa-compass"></i>
+                    <a href="https://invenio.readthedocs.io/en/latest/community/contribution-guide.html">
+                      Contribution guide
+                    </a>
+                  </h2>
+                  <p>
+                    Learn how to contribute to the project
+                  </p>
+                </div>
+                <div class="col-sm-6">
+                  <h2>
+                    <i class="fa fa-users"></i>
+                    <a href="https://invenio.readthedocs.io/en/latest/community/governance.html">Governance</a>
+                  </h2>
+                  <p>
+                    Roles and decision making
+                  </p>
+                </div>
+              </div>
+            </div>
+            <div class="col-lg-12">
+              <div class="row">
+                <div class="col-sm-6">
+                  <h2>
+                    <i class="fa fa-language"></i>
+                    <a href="https://invenio.readthedocs.io/en/latest/community/translation-guide.html">
+                      Translator guide
+                    </a>
+                  </h2>
+                  <p>
+                    Learn how to contribute as a translator
+                  </p>
+                </div>
+                <div class="col-sm-6">
+                  <h2>
+                    <i class="fa fa-bookmark"></i>
+                    <a href="https://invenio.readthedocs.io/en/latest/community/code-of-conduct.html">
```

Comment:
Not to be changed in this PR: Would it make sense to move the code of conduct page to inveniosoftware.org under community tag? (I think it would be better UX) cc @lnielsen

Reply:
Note: the "Code of conduct" is also included in "Governance" (at the very bottom). I might also link to that for now, what do you think?